docs/Using Fleet/enroll-hosts.md

```diff
@@ -45,7 +45,7 @@ The following command creates a Fleetd installer, `.pkg` file, which adds macOS
 ```sh
 fleetctl package --type pkg --fleet-url=[YOUR FLEET URL] --enroll-secret=[YOUR ENROLLMENT SECRET]
 ```
-  >**Note:** The only configuration option required to create an installer is `--type`, but to communicate with a Fleet instance, you'll need to specify a `--fleet-url` and `--enroll-secret`
+  >**Note:** The only configuration option required to create an installer is `--type`, but to communicate with a Fleet instance, you'll need to specify a `--fleet-url` and `--enroll-secret`. However, when those flags are not set, fleetd will log errors. 
```

Comment:
I'd move the "However" to make it clearer that this is a caveat to there only being 1 required configuration option:
```suggestion
  >**Note:** The only configuration option required to create an installer is `--type`. However, to communicate with a Fleet instance, you'll need to specify a `--fleet-url` and `--enroll-secret`. When those flags are not set, fleetd will log errors. 
```